vowpalwabbit/core/src/parse_args.cc

```diff
@@ -607,6 +629,10 @@ void parse_feature_tweaks(options_i& options, VW::workspace& all, bool interacti
       .add(make_option("ignore_linear", ignore_linears)
                .keep()
                .help("Ignore namespaces beginning with character <arg> for linear terms only"))
+      .add(make_option("ignore_features_dsjson_experimental", ignore_features_dsjson)
```

Comment:
if there is a plan to add it to other parsers, do we still want to call it "*_dsjson_*" (and rename later)? Maybe generic name + "not supported yet" error (for other parsers) is better?

Reply:
regardless it seems like we will have to rename it later, either by removing the "not supported yet" or the "dsjson". Once it is supported for all parsers I want the flag to be --ignore_features so all we have to do is remove the suffix. 

I dont think there is a plan to add it to all the parsers at this time, but rather refactor the parsing and add support for callback functions in the parsers. @jackgerrits will be able to elaborate more about the plan.

Reply:
I didn't mean to put "not supported" to option name, but to call option "ignore_features_experimental", with runtime error / exception raised if you are trying to call --ignore_features_experimental without --dsjson. Now it seems confusing - we have 4 data formats: json, dsjson, text, flatbuffers.
1) Is this feature supported for dsjson only (but not for json)?
2) How are we going to call it if we add plain text implementation, but not flatbuffers yet?

Reply:
Given it's experimental we can change it as much as we need, so we shouldnt worry too much about the name at the moment. When it's done and supported by all parsers then `--ignore_features` seems good. For now I think a big ugly and specific name is good

Reply:
I see, makes sense.